README.md

```diff
@@ -72,119 +157,58 @@ redis>
-    - [Documentation: Count-min sketch](https://redis.io/docs/latest/develop/data-types/probabilistic/count-min-sketch/)
-
-1. **t-digest**  (*)
-    - t-digest is a probabilistic data structure used for estimating which fraction / how many values in a data stream are smaller than a given value, which value is smaller than p percent of the values in a data stream, or what are the smallest/largest values in the data stream.
-    - [Documentation: t-digeset](https://redis.io/docs/latest/develop/data-types/probabilistic/t-digest/)
-
-1. **Time series**  (*)
-    - a time series allows storing and querying timetagged data points (samples).
-    - [Documentation: Time series quick start](https://redis.io/docs/latest/develop/data-types/timeseries/quickstart/)
-
-1. **Pub/Sub**
-    - Pub/Sub (short for publish/subscribe) is a lightweight messaging capability. Publishers send messages to a channel, and subscribers receive messages from that channel.
-    - [Documentation: Redis streams](https://redis.io/docs/latest/develop/interact/pubsub/)
-
-1. **Stream**
-    - A stream is a data structure that acts like an append-only log. Each stream entry consists of name-value string pairs, similar to a hash. Streams support multiple consumption strategies, where consumers can pop entries, consume entries by range, or listen to entries. Consumer groups allow multiple workers to retrieve different stream entries in order to scale message processing.
-    - [Documentation: Redis streams](https://redis.io/docs/latest/develop/data-types/streams/)
-
-1. **Transactions**
-    - A transaction allows the execution of a group of commands in a single step. A request sent by another client will never be served in the middle of the execution of a transaction. This guarantees that the commands are executed as a single isolated operation.
-    - [Documentation: Transactions](https://redis.io/docs/latest/develop/interact/transactions/)
-
-1. **Programmability**
-    - Users can upload and execute Lua scripts on the server. Scripts can employ programmatic control structures and use most of the commands while executing to access the database. Because scripts are executed on the server, reading and writing data from scripts is very efficient. Functions provide the same core functionality as scripts but are first-class software artifacts of the database. Redis manages functions as an integral part of the database and ensures their availability via data persistence and replication.
-    - [Documentation: Scripting with Lua](https://redis.io/docs/latest/develop/interact/programmability/eval-intro/)
-    - [Documentation: Functions](https://redis.io/docs/latest/develop/interact/programmability/functions-intro/)
-
-Items marked with (*) require building with `BUILD_WITH_MODULES=yes`.
+**Important:** Features marked with an asterisk (\*) require Redis to be compiled with the `BUILD_WITH_MODULES=yes` flag when [building Redis from source](#build-redis-from-source)
+
```

Comment:
We spent a lot of time on the descriptions of the different data types, and I find it hard to digest all these changes.
I would prefer to see a more minimal edit here. 
I also see that you switched to plural. Why do you think this is better?

Reply:
The diff makes it seem as though there are more text changes than there actually are. I mostly adjusted the format of the list to be consistent with other lists in the README. I kept the descriptions mostly the same, but tried to add use-cases for as many data types/capabilities as I could (consistent with the docs).

I switched to plural to be consistent with the docs navigation.

Reply:
One problem with plurals is that it is not always clear whether the description applies to a single data structure or a group. Anyway, there are some additional edits I want to do here, so we can merge it for now.